### PR TITLE
Moved test crob-job to base and updated image tag

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Moved test cron-job to base, and updated image tag
The test crobjob has suspend set to true
Related - #423

### CODE REVIEW

File: graph-refresh/base/cronjob.yaml [hunk 1]
```diff
@@ -26,29 +26,32 @@ spec:
...
                   valueFrom:
                     configMapKeyRef:
                       key: infra-namespace
                       name: thoth
+                - name: THOTH_GRAPH_REFRESH_SOLVER
+                  value: "1"
+                - name: THOTH_GRAPH_REFRESH_REVSOLVER
+                  value: "0"
+                - name: THOTH_GRAPH_REFRESH_SECURITY
+                  value: "0"
```
Comment (pacospace) [inline graph-refresh/base/cronjob.yaml:54]: security up as well please :)

**Review by pacospace — COMMENTED** [review]
One small comment, otherwise great!!

**Comment by harshad16** [comment]
Please remove all the rest in spec from https://github.com/thoth-station/thoth-application/blob/master/graph-refresh/overlays/test/cronjob.yaml , just keep suspend: true.

File: graph-refresh/base/cronjob.yaml (hunk 1 above)
Comment (saisankargochhayat) [reply graph-refresh/base/cronjob.yaml]: Thanks I have security to 1.

**Comment by saisankargochhayat** [comment]
> Please remove all the rest in spec from https://github.com/thoth-station/thoth-application/blob/master/graph-refresh/overlays/test/cronjob.yaml , just keep suspend: true.

Thanks, I have changed, PTAL :smile:

File: graph-refresh/overlays/test/cronjob.yaml [hunk 2]
```diff
@@ -7,122 +7,4 @@ metadata:
...
-                  port: 80
-          volumes:
-            - name: secrets
-              secret:
-                secretName: "kafka"
-                items:
-                  - key: kafka_ca.crt
-                    path: kafka_ca.crt
-          restartPolicy: OnFailure
+  suspend: true
```
Comment (harshad16) [inline graph-refresh/overlays/test/cronjob.yaml:10]: please fix the new line issue here

File: graph-refresh/overlays/stage/cronjob.yaml [hunk 3]
```diff
@@ -4,4 +4,4 @@ apiVersion: batch/v1beta1
 metadata:
   name: graph-refresh
 spec:
-  suspend: false
+  suspend: false
```
Comment (harshad16) [inline graph-refresh/overlays/stage/cronjob.yaml:7]: please remove the newline here as well,
no changes were needed in this file.

File: graph-refresh/overlays/test/cronjob.yaml (hunk 2 above)
Comment (saisankargochhayat) [reply graph-refresh/overlays/test/cronjob.yaml]: thanks, added precommit fixes

File: graph-refresh/overlays/stage/cronjob.yaml (hunk 3 above)
Comment (saisankargochhayat) [reply graph-refresh/overlays/stage/cronjob.yaml]: removed :)

**Review by harshad16 — APPROVED** [review]
lgtm
thanks

**Pull request merged**